rfcs/20250314-graph-api-bottom-right-causal-mask/README.md

```diff
@@ -0,0 +1,294 @@
+# Support Implicit Causal Mask in SDPA in Graph API
+
+## Background
+
+Scaled Dot-Product Attention (SDPA), introduced in [1], is the core operation in
+Transformer blocks and serves as the backbone for numerous language and generative
+models (e.g., BERT, Stable Diffusion, GPT, DeepSeek).
+
+oneDNN supports SDPA and its optimization through Graph API by defining an SDPA
+graph, partitioning the graph, and optimizing the underlying kernels. The
+floating-point SDPA — supporting f32, bf16, or f16 — is defined as illustrated below:
+
+![oneDNN Graph SDPA pattern](images/onednn_graph_sdpa.png)
+
```

Comment:
Does brown module mean optional ? If there are two colors of modules in the flowchart, better to add annotations explaining what each color represents. Same comments for the other pictures.